app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -1,2 +1,20 @@
 class ManageIQ::Providers::IbmCloud::VPC::CloudManager::Vm < ManageIQ::Providers::CloudManager::Vm
+  # https://cloud.ibm.com/apidocs/vpc#get-instance
+  POWER_STATES = {
+    "running"       => "on",
+    "failed"        => "off",
+    "paused"        => "suspended",
```

Comment:
There's a couple of these states that you could map more directly and still get [icons](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/lib/gtl_formatter.rb#L19-L41):
```
'paused'     => 'paused',
'pausing'    => 'paused',
'restarting' => 'reboot_in_progress',
```

Reply:
Thanks Jay - will add these.